base/src/main/java/com/smartdevicelink/proxy/rpc/ShowConstantTbtResponse.java

```diff
@@ -46,19 +46,27 @@
  */
 public class ShowConstantTbtResponse extends RPCResponse {
 
+    /**
```

Comment:
@kostyaBoss I would delete the comment block on lines 49-51.  The idea is that nothing should be changed in this file.